articles/industry-4.0-and-cybersecurity/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/industry-4.0-and-cybersecurity/
+title: Industry 4.0 and Cybersecurity
+description: An introduction to Industry 4.0 through the enabling technologies, including fundamental principles of security, cybersecurity challenges, threats and insinuations.
+author: collins-agesa
+date: 2020-09-06T00:00:00-08:00
+topics: [Cybersecurity]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/industry-4.0-and-cybersecurity/hero.jpg
+    alt: bulma image example
+---
+Many technologies have emerged in recent times. These technologies include the Internet of Things, Big Data Analytics and Cyber-Physical Systems. They have enabled new and impactful initiatives such as Industry 4.0. this initiative is revolutionizing the manufacturing industry. Yet, the cybersecurity challenges associated with the technology have never been greater.
+<!--more-->
+
+## Industry 4.0
+
+Industry 4.0 had been a concept of German origin as a government initiative.  Dubbed as the fourth industrial revolution, it was first introduced as Industrie 4.0 in 2011. The general goal was to improve competitiveness of the manufacturing industry. It was also to advance the new generation of smart manufacturing systems. Enabling technologies played a big part to support this goal. These technologies include:
+
+***Internet of Things (IoT):** a network of interconnected objects. Objects are physical devices and are billions in number globally, through the internet. IIoT refers to Industrial Internet of Things.
+
+***Cyber-Physical Systems:** integrations of physical processes with computation and networking processes. Key for automation and monitoring.
+
+***Smart Factory:** advanced robotics, machine learning, additive manufacturing for rapid prototyping
+
+***Cloud:** (Big Data Analytics) for storage as well as computation among others.
+
+Depending on perspective, Industry 4.0 is a result of the Industrial Internet of Things. It is revolutionizing the manufacturing sector with innumerable benefits. Increase in efficiency, productivity, flexibility, agility are a few. Others include collaborative operation and sharing. The outcome includes lowering overhead, leading to increase profitability. Improved customer experience and facilitation of greater innovation and creativity is also noted.
+
+## Fundamental Principles of Security
+
+It is important to understand the Industry 4.0 cybersecurity landscape. We do this by first identifying the base principles of any security system. These outline the main objectives of information security.
+
+### Confidentiality
+
+This is sustaining the secrecy of the flow of information. In this context, throughout the manufacturing system value chains.
+
+Industry 4.0 systems are complex yet flexible. This provides many points where attackers could intercept and tap information.
```

Comment:
How are they complex yet flexible? How do attackers intercept information? Where is this information even stored in the context of Industry 4.0?